packages/origin-device-nvenergy-api/src/main.ts

```diff
@@ -14,6 +14,6 @@ async function bootstrap() {
   const document = SwaggerModule.createDocument(app, options);
   SwaggerModule.setup('api', app, document);
 
-  await app.listen(3000);
+  await app.listen(3001);
```

Comment:
This is unnecessary, the `main.ts` can be removed from nvenergy-api npm package. This is not meant to be run separately, at least not by default.